### PR TITLE
Enable throttling for JSONP API requests.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: remo/api/__init__.py [hunk 1]
```diff
@@ -10,3 +13,27 @@ def __init__(self, control={'no_cache': True}, *args, **kwargs):
 
     def cache_control(self):
         return self.control
+
+
+class RemoThrottleMixin(object):
```
Comment (akatsoulas) [inline remo/api/__init__.py]: Please add docstrings in both classes.

**Comment by akatsoulas** [comment]
Shouldn't you log the clients in order to determine which are going to be throttled?

**Comment by akatsoulas** [comment]
tested locally and looks good. Let's merge this and test it further in dev
r+ :bear:

**Pull request merged**